controllers/feeds/getOneFeedChat.js

```diff
@@ -158,7 +158,9 @@ const getOneFeedChatService = async (req, res) => {
       actor.id = null;
     }
 
-    const titlePost = getMessage(data.results[0]);
+    const titlePost = options?.getNewMediaMessage
+      ? getMessage(data.results[0])
+      : data.results[0].message || '';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The ternary operation here could potentially lead to an error if `data.results[0]` is undefined or null. It would be safer to add checks for these conditions before accessing properties on `data.results[0]`.

```diff
- const titlePost = options?.getNewMediaMessage
-   ? getMessage(data.results[0])
-   : data.results[0].message || '';
+ const titlePost = options?.getNewMediaMessage && data.results[0]
+   ? getMessage(data.results[0])
+   : data.results[0]?.message || '';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->